include/cbind.h

```diff
@@ -54,7 +52,7 @@ void delete_popup ( IDSIMMODEL* model, POPUPID id );
 void dump_to_debug_popup ( IDEBUGPOPUP* popup, const uint8_t* buf, uint32_t offset, uint32_t size );
 void print_to_debug_popup ( IDEBUGPOPUP* popup, const char* message );
 void repaint_memory_popup ( IMEMORYPOPUP* popup );
-void set_memory_popup ( IMEMORYPOPUP* popup, uint32_t offset, void* buffer, size_t size );
+void set_memory_popup ( IMEMORYPOPUP* popup, uint32_t offset, void* buffer, uint32_t size );
```

Comment:
You should change the corresponding cbind.c. The comment sections still refers to the argument as size_t

Reply:
merged. I would recommend you to delete this PR and create a new one just for Makefile changes. One fix - one PR.